src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -24,11 +24,11 @@ public void parse_emptyArg_throwsParseException() {
     public void parse_validArgs_returnsFindCommand() {
         // no leading and trailing whitespaces
         FindCommand expectedFindCommand =
-                new FindCommand(new NameContainsKeywordsPredicate(Arrays.asList("Alice", "Bob")));
-        assertParseSuccess(parser, "Alice Bob", expectedFindCommand);
+                new FindCommand(new TitleContainsKeywordsPredicate(Arrays.asList("Amy", "Bob")));
```

Comment:
What's the rationale of changing it to Amy?

Reply:
changes reverted